src/ape/api/projects.py

```diff
@@ -203,6 +206,15 @@ def cached_manifest(self) -> Optional[PackageManifest]:
         """
         return _load_manifest_from_file(self._target_manifest_cache_file)
 
+    def __getattr__(self, item: str) -> "ContractContainer":
+        manifest = self.extract_manifest()
+        if hasattr(manifest, item):
+            from ape.contracts import ContractContainer
```

Comment:
What would be the issues in flipping this cycle so that we don't have to import it internally here? I see `BaseAddress` and `Address` used in `ape.contracts`

Perhaps for another PR refactoring that out into `ape.types` somehow

Reply:
I was also wondering if it would make sense to have access to the partial off of the manager-base-mixin. That way, you can create contract containers anywhere.